packages/dmn-js-drd/src/features/drill-down/DrillDown.js

```diff
@@ -71,7 +71,7 @@ export default class DrillDown {
   addOverlay(element, className) {
     const html = domify(`
       <div class="drill-down-overlay">
-        <span class="${className}"></span>
+        <button type="button" class="${className}"></button>
```

Comment:
The styling is not quite there yet. I'd rather keep the old look for now.

Reply:
@barmac can you explain a bit more about the styling? I guess you mean that the previous blue background is missing on the new button element.

**Before:**
<img width="59" alt="Screenshot 2023-11-28 at 15 20 35" src="https://github.com/bpmn-io/dmn-js/assets/6731968/42fcbcae-70de-4ae0-afab-ba4abe57475e">


**After:**

<img width="68" alt="Screenshot 2023-11-28 at 15 19 50" src="https://github.com/bpmn-io/dmn-js/assets/6731968/51f39602-29ca-45b4-a733-dcf3e97fc749">

Reply:
I've pushed a new commit to match the styling that was there previously on the span.
What do you think? __https://github.com/bpmn-io/dmn-js/pull/802/commits/34041ad1349528a740d1e369514069831a915292

_Without focus_

<img width="107" alt="Screenshot 2023-11-28 at 15 42 03" src="https://github.com/bpmn-io/dmn-js/assets/6731968/8a3772cd-1d68-41f0-98ed-5450424105f4">

_With focus_

<img width="865" alt="Screenshot 2023-11-28 at 15 41 50" src="https://github.com/bpmn-io/dmn-js/assets/6731968/f00f19ea-864c-4be2-b68a-65902a5e9e5b">

Reply:
What I meant is that if we switch to standard button look, we change the visuals significantly. On the other hand, we need **some** way to indicate focus.

Reply:
@barmac I've pushed [another commit](https://github.com/bpmn-io/dmn-js/pull/802/commits/e3d7e1484280e0d21e32ddf1061c4d9a8722cd6b) so the focus style is more apparent on the drill down button. 

Do you see some specific style difference between original and new style as described below?
If so I can try to adjust it further to match the expectation.


For comparison this is the _original span style on focus_, i.e, no focus focus style applied just span with blue background:

<img width="525" alt="Screenshot 2023-12-04 at 11 27 24" src="https://github.com/bpmn-io/dmn-js/assets/6731968/7587bd88-2b6c-412f-915b-dccf8cc78b80">

Whereas, this is the latest commit blue button which IMO looks the same as original span style, _but with the browser focus style being applied_:


_Before focus (default style)_


<img width="768" alt="Screenshot 2023-12-04 at 11 24 34" src="https://github.com/bpmn-io/dmn-js/assets/6731968/7b7cbba9-9ab6-4fe2-8e5b-def76438138a">


_On focus (Dark blue focus style applied by browser)_

<img width="713" alt="Screenshot 2023-12-04 at 11 24 40" src="https://github.com/bpmn-io/dmn-js/assets/6731968/668cbecf-2a1a-4f13-bd30-9498d69dc684">

Reply:
Thanks, it looks better now. I will tweak it a bit and then merge this PR :)

Reply:
Great, looking forward to the next release then :)